pkg/config/evaluation.go

```diff
@@ -73,6 +74,23 @@ func (r *RunConfigV1) evaluateConditionsV1(config *Config, filters map[string]St
 					}
 				}
 			}
+
+			// TODO: PART 1 : if explicit activation/de-activation is available should notActiveConditions be checked ?
```

Comment:
move this into above `else` block?

Currently a `notActive` condition would overrule explicit activation?
Do we want to keep this for now knowing the Fortify topic or rather make sure that explicit activation/deactivation is always considered?

Reply:
once we anchor fortify to another parameter, we can move it , what are your thoughts ?